### ISSUE TITLE
NumberControl title should be optional

### ISSUE THREAD

**Issue by samreid** [phetsims/scenery-phet#512]
From https://github.com/phetsims/scenery-phet/issues/508#issuecomment-507062895

> I cannot figure out how to get NumberControl to not show a title. Some of the examples in https://github.com/phetsims/scenery-phet/issues/508#issuecomment-505693401  have tweakers but no title.

@pixelzoom replied

> Provide a custom layout function via `options.layoutFunction`.   For an example of a NumberControl with no title, see `temperatureLayoutFunction` in `GAS_PROPERTIES/InjectionTemperatureAccordionBox`. Yes, it's a bit inconvenient/wasteful to create a title that isn't used.  If that's something you want to pursue, create an issue to move `title` from a constructor parameter to an `option`.  (It would also be more general for `title` to be `{Node}` instead of `{string}`.)

I'm not sure of the best time to work on this, perhaps next time we develop a simulation (or revisit a pre-existing simulation) that has a NumberControl without a title.  Assigning to @ariel-phet for now so he's aware of this request, and can comment on timing, priority, scheduling and delegation.

**Comment by pixelzoom** [comment]
I'll do this.

**Comment by pixelzoom** [comment]
This is a much bigger can of worms than I'd guessed.  

First I looked at changing the NumberControl constructor signature from `{string} title` to `{Node} titleNode`. This would be a lot of work, with 36 uses in 10 sims, and would involve requiring the sims to construct the `{Node} titleNode`.   100% of those sims currently use strings.   So until some sim needs a non-string title, I recommend that we leave this "as is". 

Next I looked at making the title optional.  This is even more work.  100% of the predefined layout functions rely on having a title, as does the API for the signature of the layout function:

`function layoutFunction( titleNode, numberDisplay, slider, leftArrowButton, rightArrowButton )`

So if you don't want a title, I think the best best is still:

`new NumberControl( '', numberProperty, numberRange, options )`  

... with a custom `options.layoutFunction` that ignores its `titleNode` arg.  And if this becomes common, we can add a predefined layout function that ignore the title.  

So based on the cost:benefit for this change, my recommendation is to do nothing.

Back to @ariel-phet.
